commcare_connect/opportunity/views.py

```diff
@@ -973,6 +951,9 @@ def approve_visit(request, org_slug=None, pk=None):
         user_visit.save()
         update_payment_accrued(opportunity=user_visit.opportunity, users=[user_visit.user])
 
+    if request.htmx:
```

Comment:
todo: remove other usages of this view, then this check is unnecessary

Reply:
Fixed. [e61340a](https://github.com/dimagi/commcare-connect/pull/571/commits/e61340ad278a790d429dce2edb810cb344f08302)

Removed references for unused views, templates and updated links to point to newly added views.